src/modules/core/components/Permission/Permission.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+import { MessageDescriptor, useIntl } from 'react-intl';
+import Icon from '~core/Icon';
+import { Tooltip } from '../Popover';
+import styles from './Permission.css';
+import { permissionsObject } from './permissions';
+
+interface Props {
+  /** Permission name */
+  permission:
```

Comment:
Please replace these types with the official ones imported from `colonyJS`
```js
import { ColonyRole } from '@colony/colony-js'
```
And `ColonyRole` will contain all the roles we need:

![Screenshot from 2020-11-17 21-09-04](https://user-images.githubusercontent.com/1193222/99435732-341a6b80-2919-11eb-9c5f-b68c56fdb83c.png)